learningresources/views.py

```diff
@@ -42,6 +52,11 @@ def create_repo(request):
         form = RepositoryForm(data=request.POST)
         if form.is_valid():
             repo = form.save(request.user)
+            assign_user_to_repo_group(
```

Comment:
I'm biased against signals because they're "magical." If you're reading the code to figure out how it works, or trying to track down a bug, you're not getting the full story.

They're useful for things like auditing and attaching behavior to models outside your codebase, but if its our own code, I'd prefer to avoid signals.